code/data_structures/src/stack/sort_stack/sort_stack.js

```diff
@@ -0,0 +1,55 @@
+var Stack = function (initStack = [])
+{
+    this.stack = initStack;
+    this.top = initStack.length;
+    
+    this.push = function (value) {
+        this.stack[this.top++] = value;
+    }
+
+    this.size = function() {
+        return this.top;
+    }
+
+    this.peek = function() {
+        return this.stack[this.top - 1];
+    }
+
+    this.pop = function () { 
+        if (this.top === 0) {
+            return "Stack is Empty";
```

Comment:
Don't mix and match return types (unless undefined). Either use an empty `return;` or throw an exception.